### ISSUE TITLE
Clarification of service parameter self-description

### ISSUE THREAD

**Issue by gpdf** [ivoa-std/DALI#42]
Currently the document has many (useful!) paragraphs of the form:

> Floating point interval values serialised in VOTable or service parameters must have the following metadata in the `FIELD` element: `datatype="double"` or `datatype="float", arraysize="2", xtype="interval"`.

I believe the intent of the mention of "service parameters" here is intended to provide guidance for how to describe a service parameter in a service descriptor, either for an onward link to a related service, or in a [service self-description](https://www.ivoa.net/documents/DataLink/20231215/REC-DataLink-1.1.html#tth_sEc4.4).  

But in a service descriptor, the enclosing element is a `PARAM`, not a `FIELD`.  For clarity for implementers, I recommend that we find a way to state this more correctly.  One option might be:

> Floating point interval values serialised in VOTable (or defined as parameter values in service descriptors) must have the following metadata in the corresponding `FIELD` (or `PARAM`) element: `datatype="double"` or `datatype="float", arraysize="2", xtype="interval"`.

**Comment by pdowler** [comment]
Agreed. Do you want to create a PR for all the places this improvement could be done?

**Comment by gpdf** [comment]
Yes, I'm up for that.

**Comment by pdowler** [comment]
Sounds good.

**Comment by pdowler** [comment]
Since this is still sitting and I just fixed #41 I will also address this issue in that same PR.
